unleash/src/main/java/com/silvercar/unleash/repository/JsonToggleCollectionDeserializer.java

```diff
@@ -41,7 +41,19 @@ static ToggleCollection deserializeVersion0(JsonElement rootElement, JsonDeseria
 
         JsonArray features = rootElement.getAsJsonObject().getAsJsonArray("features");
 
-        features.forEach(elm -> {
+        // TODO: Use Kotlins method after Kotlion conversion - original line below
+//        features.forEach(elm -> {
+//            JsonObject featureObj = elm.getAsJsonObject();
+//
+//            String name = featureObj.get("name").getAsString();
+//            boolean enabled = featureObj.get("enabled").getAsBoolean();
+//            String strategyName = featureObj.get("strategy").getAsString();
+//            Map<String, String> strategyParams = context.deserialize(featureObj.get("parameters"), PARAMS_TYPE);
+//
+//            ActivationStrategy strategy = new ActivationStrategy(strategyName, strategyParams);
+//            featureToggles.add(new FeatureToggle(name, enabled, singletonList(strategy), emptyList()));
+//        });
```

Comment:
@FernandoX7 Do we need to check this in?

Reply:
I left them in with TODO's because when converted to kotlin - you can then paste the commented code and most of the time it'll auto-convert it perfectly.